zoom_monitor.py

```diff
@@ -150,92 +172,74 @@ def _open_participants_list(self):
-                    time.sleep(1)  # The DOM isn't set up; wait a little longer
-                    break  # Go to the next overall attempt
-
-                # Now that we've clicked the participants list without raising
-                # an exception, wait until it shows up.
-                self._wait_for_element(
-                    By.CLASS_NAME, "participants-wrapper__inner")
-                self._logger.info("participants list opened")
-                return  # Success!
+            button = self._find_participants_button()
+            try:
+                # Clicking on the participants list only selects the button.
+                # As a small clue: if a human clicks on it, the mouse-down
+                # selects the button while the mouse-up opens the participants
+                # list. Double-clicking on it seems to work okay (maybe the
+                # second click implicitly creates a mouse-up on the first one).
+                # If the button is already selected, only one click is needed.
+                button.click()
+                if not selected:
+                    button.click()  # Channeling our inner grandma
+                self._logger.debug("participants list clicked")
+            except ElementClickInterceptedException:
+                self._logger.debug("DOM isn't set up; wait and try again")
+                time.sleep(1)  # The DOM isn't set up; wait a little longer
+                continue  # Go to the next attempt
+
+            # Now that we've clicked the participants list without raising
+            # an exception, wait until it shows up.
+            self._wait_for_element(
+                By.CLASS_NAME, "participants-wrapper__inner")
+            self._logger.info("participants list opened")
+            return  # Success!
 
         # If we get here, none of our attempts opened the participants list.
         raise ElementClickInterceptedException(
             f"Could not open participants list after {attempt + 1} attempts")
 
-    def _wait_for_element(self, approach, value):  # Helper function
-        """
-        Wait until there is at least one element identified by the approach
-        and value. If 5 seconds elapse without such an element appearing, we
-        raise an exception.
-        Return the element the first element that is found.
+    def _is_selected(self):
         """
-        WebDriverWait(self._driver, 5).until(lambda _:
-            len(self._driver.find_elements(approach, value)) != 0)
-        return self._driver.find_elements(approach, value)[0]
+        Find the participants icon using the class name.
 
-    def _checkIfMeetingEnded(self):
+        The two classes the participant icon can have are:
+        "SvgParticipantsDefault" - the default button class.
+        "SvgParticipantsHovered" - the button is already selected.
+
+        Return if the participants button is selected or not.
```

Comment:
Wording tweak: "return whether the participants button is selected." As written, until you get to the very end of the sentence it sounds like maybe we don't return from the function when the participants button isn't selected.